dev-docs/bidders/lemmadigital.md

```diff
@@ -5,8 +5,10 @@ description: Prebid Lemmadigital Bidder Adapter
 biddercode: lemmadigital
 media_types: video,banner
 pbjs: true
+pbs: true
 schain_supported: true
 floors_supported: true
+pbs_app_supported: true
```

Comment:
while you are at it, you can add more meta information which makes vetting your adapter easier:

- multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
- safeframes_ok: true/false
- ortb_blocking_supported: true/partial/false

Reply:
Agreed, added mentioned meta info. Thanks!
[Ref commit](https://github.com/prebid/prebid.github.io/pull/4897/commits/fd9d87a97901b26c15962427861c321b6f161a08).